libcontainer/container_linux.go

```diff
@@ -767,6 +767,25 @@ func (c *linuxContainer) updateState(process parentProcess) error {
 	return json.NewEncoder(f).Encode(state)
 }
 
+func (c *linuxContainer) checkFreezer() (Status, error) {
+	path := c.cgroupManager.GetPaths()["freezer"]
+	if _, err := os.Stat(path); err != nil {
+		if os.IsNotExist(err) {
+			return Running, nil
```

Comment:
this seems a little weird to have here.  Maybe a comment explaining why the status would be running if the error is an IsNotExist error.